### PR TITLE
Merge meed 2962

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: portlets/src/main/webapp/WEB-INF/gatein-resources.xml [hunk 1]
```diff
@@ -778,6 +778,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
   <module>
     <name>ruleComponents</name>
+    <load-group>gamification-admin-connectors</load-group>
```
Comment (boubaker) [inline portlets/src/main/webapp/WEB-INF/gatein-resources.xml]: you may need to add a dedicated module to load the ruleComponents for `gamification-admin-connectors`.
The need isn't to load the `gamification-admin-connectors` each time we load `ruleComponents` isn't it ?

File: portlets/src/main/webapp/vue-app/connector-admin-settings/components/AdminConnectorSettings.vue [hunk 2]
```diff
@@ -106,15 +106,15 @@ export default {
           const filteredList = this.adminConnectorsExtensions.filter(connectorExtension => !connectors.some(item => item.name === connectorExtension.componentOptions.name)).map(item => ({
             name: item?.componentOptions?.name,
             title: item?.componentOptions?.title,
+            comingSoon: item?.componentOptions?.comingSoon,
```
Comment (boubaker) [inline portlets/src/main/webapp/vue-app/connector-admin-settings/components/AdminConnectorSettings.vue]: May be passing all options all time, WDYT ?

```suggestion
            .map(item => item?.componentOptions).filter(e => e)
```

File: portlets/src/main/webapp/WEB-INF/gatein-resources.xml (hunk 1 above)
Comment (AzmiTouil) [reply portlets/src/main/webapp/WEB-INF/gatein-resources.xml]: yes, Done!

File: portlets/src/main/webapp/vue-app/connector-admin-settings/components/AdminConnectorSettings.vue (hunk 2 above)
Comment (AzmiTouil) [reply portlets/src/main/webapp/vue-app/connector-admin-settings/components/AdminConnectorSettings.vue]: Done!

**Review by boubaker — APPROVED** [review]

**Pull request merged**